src/test/kotlin/uk/gov/dluhc/printapi/rest/GetAnonymousElectorDocumentSummaryByApplicationIdIntegrationTest.kt

```diff
@@ -0,0 +1,187 @@
+package uk.gov.dluhc.printapi.rest
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType.APPLICATION_JSON
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import uk.gov.dluhc.printapi.database.entity.DeliveryAddressType.ERO_COLLECTION
+import uk.gov.dluhc.printapi.database.entity.DeliveryAddressType.REGISTERED
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.database.entity.SourceType.ANONYMOUS_ELECTOR_DOCUMENT
+import uk.gov.dluhc.printapi.models.AnonymousElectorDocumentStatus.PRINTED
+import uk.gov.dluhc.printapi.models.AnonymousElectorDocumentSummariesResponse
+import uk.gov.dluhc.printapi.models.CertificateLanguage.EN
+import uk.gov.dluhc.printapi.models.DeliveryAddressType
+import uk.gov.dluhc.printapi.models.SupportingInformationFormat.STANDARD
+import uk.gov.dluhc.printapi.testsupport.bearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+import uk.gov.dluhc.printapi.testsupport.testdata.anotherValidEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAedContactDetails
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAnonymousElectorDocument
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildDelivery
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomGssCode
+import uk.gov.dluhc.printapi.testsupport.testdata.getVCAnonymousAdminBearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildAnonymousElector
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildAnonymousElectorDocumentSummary
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildElectoralRegistrationOfficeResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildLocalAuthorityResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildValidAddress
+import java.time.ZoneOffset
+
+internal class GetAnonymousElectorDocumentSummaryByApplicationIdIntegrationTest : IntegrationTest() {
+    companion object {
+        private const val URI_TEMPLATE = "/eros/{ERO_ID}/anonymous-elector-documents?applicationId={APPLICATION_ID}"
+        private const val APPLICATION_ID = "7762ccac7c056046b75d4bbc"
+        private const val GSS_CODE = "W06000099"
+    }
+
```

Comment:
Don't we normally have a few more std security tests?

Reply:
ControllerSpringSecurityIntegrationTest are introduced that tests for basic security tests. but i can add them in this int tests if you want?

Reply:
`ControllerSpringSecurityIntegrationTest` updated to have a test around accessing aed get endpoint